packages/synapse-react-client/src/lib/containers/MarkdownPopover.tsx

```diff
@@ -1,124 +1,105 @@
-    useState<HTMLSpanElement | null>(null)
-  const [popperElement, setPopperElement] = useState<HTMLDivElement | null>(
-    null,
-  )
-  const [arrowElement, setArrowElement] = useState<HTMLDivElement | null>(null)
-
-  const { styles, attributes } = usePopper(referenceElement, popperElement, {
-    placement: placement,
-    strategy: strategy,
-    modifiers: [
-      { name: 'arrow', options: { element: arrowElement } },
-      {
-        name: 'offset',
-        options: {
-          offset: [0, 10],
-        },
-      },
-    ],
-  })
-
-  const popover = (
-    <CSSTransition
-      in={show}
-      timeout={200}
-      classNames="fade-in-out"
-      mountOnEnter
-    >
-      <div
-        className={`Tooltip SRC-popover bootstrap-4-backport`}
-        style={{ ...style, ...styles.popper }}
-        ref={setPopperElement}
-        {...attributes.popper}
-      >
-        <Typography variant="body1">
-          <MarkdownSynapse {...contentProps} />
-        </Typography>
-        <div className="TooltipButtonContainer">
-          {actionButton && (
-            <Button
-              color="primary"
-              variant="contained"
-              onClick={() => {
-                actionButton.onClick()
-                if (actionButton.closePopoverOnClick) {
-                  setShow(false)
-                }
-              }}
-            >
-              {actionButton.content}
-            </Button>
-          )}
-          {showCloseButton && (
-            <Button variant="outlined" onClick={() => setShow(false)}>
-              Close
-            </Button>
-          )}
-        </div>
-        <div
-          className="arrow"
-          data-popper-arrow
-          ref={setArrowElement}
-          style={styles.arrow}
-        />
-      </div>
-    </CSSTransition>
+  const content = (
+    <Box sx={{ padding: '20px' }}>
+      <Typography variant="body1" marginBottom={2}>
+        <MarkdownSynapse {...contentProps} />
+      </Typography>
+      <Box sx={buttonBoxSx}>
+        {actionButton && (
+          <Button
+            color={actionButton.color || 'primary'}
+            variant={actionButton.variant || 'contained'}
+            onClick={() => {
+              actionButton.onClick()
+              if (actionButton.closePopoverOnClick) {
+                setShow(false)
+              }
+            }}
+          >
+            {actionButton.content}
+          </Button>
+        )}
+        {showCloseButton && (
+          <Button variant="outlined" onClick={() => setShow(false)}>
+            Close
+          </Button>
+        )}
+      </Box>
+    </Box>
   )
 
   return (
-    <>
-      <span
-        role="button"
-        className="PopoverContainer bootstrap-4-backport"
-        onClick={() => setShow(val => !val)}
-        ref={setReferenceElement}
+    <Box
+      className={'PopoverContainer'}
+      sx={{ display: 'inline-block', cursor: 'pointer' }}
+    >
```

Comment:
Class is used for positioning the HelpPopover info icon in DownloadCartPage -- 

main | feature
:---:|:---:
<img width="1391" alt="main_HelpPopover_DownloadCartPage" src="https://user-images.githubusercontent.com/26949006/235801037-825df2ff-c264-428b-9f2e-6e4a7009ba63.png"> | <img width="1386" alt="feature_HelpPopover_DownloadCartPage" src="https://user-images.githubusercontent.com/26949006/235801039-93b63f7d-1e5b-478a-812d-27698c7b4e95.png">